docs/src/app/components/pages/components/icon-buttons.jsx

```diff
@@ -135,17 +137,18 @@ export default class IconButtonsPage extends React.Component {
         </Paper>
 
         <CodeExample code={Code}>
-          <IconButton iconClassName="muidocs-icon-custom-github" tooltip="bottom-right" tooltipPosition = "bottom-right" />
-
-          <IconButton iconClassName="muidocs-icon-custom-github" tooltip="bottom-center" tooltipPosition = "bottom-center" />
-
-          <IconButton iconClassName="muidocs-icon-custom-github" tooltip="bottom-left" tooltipPosition = "bottom-left" />
-
-          <IconButton iconClassName="muidocs-icon-custom-github" tooltip="top-right" tooltipPosition = "top-right" />
-
-          <IconButton iconClassName="muidocs-icon-custom-github" tooltip="top-center" tooltipPosition = "top-center" />
-
-          <IconButton iconClassName="muidocs-icon-custom-github" tooltip="top-left" tooltipPosition = "top-left" />
+          <IconButton iconClassName="muidocs-icon-custom-github" tooltip="bottom-right"
+            tooltipPosition="bottom-right" />
```

Comment:
There is a eslint rule for this. Could be a good idea to enforce it.
But I would say it's for later :grin: